patterns/hero-product-chessboard.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Title: Hero Product Chessboard
+ * Slug: woocommerce-blocks/hero-product-chessboard
+ * Categories: WooCommerce
+ */
```

Comment:
Not sure if it's really useful for this pattern since it's not designed for a specific template part 🤔 
I'll merge and if necessary we can always add it later.
Thanks 😄